packages/@o3r/schematics/src/utility/index.ts

```diff
@@ -1,4 +1,5 @@
 export * from './ast';
+export * from './builder';
```

Comment:
this doesn't look like the autofix of a regexp

Reply:
I think he get a conflict issue while rebasing the different PRs with auto-fix 
I will fix it in the manual fix if that the only comment

Reply:
There it expose twice `builder`

Reply:
Yes, added in a conflict resolving